go.mod

```diff
@@ -18,13 +20,13 @@ require (
 )
 
 require (
-	cloud.google.com/go v0.114.0 // indirect
+	cloud.google.com/go v0.116.0 // indirect
```

Comment:
Similarly, there are some dependency upgrades I would try to avoid bumping in PR for a different feature

Reply:
I think a wrong command and made a mess on the go.mod. I copied the existing one and only ran mod tidy for it to made the necessary updates